src/containers/feedOfClaim/index.tsx

```diff
@@ -32,6 +32,7 @@ import { AddCircleOutlineOutlined } from '@mui/icons-material'
 import MainContainer from '../../components/MainContainer'
 import { checkAuth } from '../../utils/authUtils'
 import Redirection from '../../components/RedirectPage'
+import { sleep } from '../../utils/promise.utils'
 
 const CLAIM_ROOT_URL = 'https://live.linkedtrust.us/claims'
```

Comment:
this is hardcoded to the prod server, so new claims on dev are never going to show up here, this should be an env variable